src/filters/index.js

```diff
@@ -63,20 +63,23 @@ export default {
   /**
    * 12345 => $12,345.00
    *
-   * @param {String} sign
+   * @param {String} currency
+   * @param {Number} places
    */
 
-  currency (value, currency) {
+  currency (value, currency, places) {
```

Comment:
I would use `precision` as the terminology is a bit more precise.  As you are defining the accuracy of the output.